src/app/pages/class/services/class.graphql.ts

```diff
@@ -52,6 +57,14 @@ export const QUERY_GET_CLASS_BY_NAME = gql`
         }
       }`;
 
+// export const QUERY_GET_CLASS_BY_LESSON_TITLE = gql`
```

Comment:
Same here and below, can you please remove comments?

Reply:
done